src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -138,10 +138,12 @@ class Decimal extends React.Component {
   };
 
   removeDelimiters = (value) => {
-    /* If a value consists of only commas or only letters and commas, we want to treat that 
+    /* If a value is passed in that is a number but has too many delimiters, we want to handle this
+       value without formatting it or removing delimiters. We also want to consider that,
+       if a value consists of only commas or only letters and commas, we want to treat that 
        value in the same way as if the value was NaN. We want to pass this value to the 
        formatValue function, before the commas can be removed. */
-    if (value.match(/^[a-zA-Z,\W_]*$/)) {
+    if (value.match(/([^A-Za-z0-9]{2,})/)) {
```

Comment:
The delimiter isn't always `,`, it depends on the users locale.

Reply:
This will match any special (non-alphanumeric) characters so I think just the comment needs updating?